Readme.md

```diff
@@ -443,6 +444,22 @@ observableService("Foo", 0)
     .disposed(by: disposeBag)
 ```
 
+#### zipWith
+
+Convenience operator for writing:
```

Comment:
I don't think this is good for "documentation". 

I'd personally prefer you'd pass the method description (e.g. merges all specific observables ... etc), so it's actually descriptive of the operator.

Reply:
updated